src/core_atmosphere/physics/mpas_atmphys_driver.F

```diff
@@ -270,14 +271,21 @@ subroutine physics_driver(domain,itimestep,xtime_s)
 
     !call to land-surface scheme:
     if(config_lsm_scheme .ne. 'off') then
-       call allocate_lsm(config_frac_seaice)
+       call allocate_lsm
 !$OMP PARALLEL DO
        do thread=1,nThreads
           call driver_lsm(itimestep,block%configs,mesh,diag_physics,sfc_input, &
                           cellSolveThreadStart(thread), cellSolveThreadEnd(thread))
        end do
 !$OMP END PARALLEL DO
-       call deallocate_lsm(config_frac_seaice)
+       call deallocate_lsm
+
+       call allocate_seaice
+       do thread=1,nThreads
```

Comment:
So that the `driver_seaice` call can be parallelized with OpenMP, I think we should add directives around this `do` loop so that the code looks like:
```fortran
!$OMP PARALLEL DO
       do thread=1,nThreads
          call driver_seaice(block%configs,diag_physics,sfc_input, &
                             cellSolveThreadStart(thread),cellSolveThreadEnd(thread))
       enddo
!$OMP END PARALLEL DO
```
(essentially following what was done for the thread loop around the call to `driver_lsm`.

Reply:
As you suggested, I added OpenMP directives around the loop that calls the main seaice driver. Thanks for the comment.